esti/system_test.go

```diff
@@ -73,6 +73,14 @@ func setupTest(t *testing.T) (context.Context, logging.Logger, string) {
 	return ctx, logger, repo
 }
 
+func tearDownTest(t *testing.T, repoName string) {
+	ctx := context.Background()
```

Comment:
We should have a flag / env variable that should be disabled by default and we can toggle to perform the teardown. We are purposely not cleaning up tests for debugging